tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -208,7 +208,7 @@ def access_string_vhdl(dut):
     tlog = logging.getLogger("cocotb.test")
```

Comment:
I'm quite doubtful that this tests actually works, either before or after. In particular, line 260 looks like nonsense to me. Could someone test if for me in a vhdl simulator?